### ISSUE TITLE
Inconsistent underscores false positive?

### ISSUE THREAD

**Issue by gilescope** [rust-lang/rust-clippy#6836]
Ok so this is a fun one. `cargo fmt` doesn't understand astetics of verticals alas ( https://github.com/rust-lang/rustfmt/issues/4710 ), but with the odd trailing `_` you can fool `cargo fmt` into doing the right thing and making things beautiful:

https://github.com/mimblewimble/grin/pull/3572/files

But clippy's not happy with inconsistent underscores. ( https://rust-lang.github.io/rust-clippy/master/index.html#inconsistent_digit_grouping )

Is there a better solution than a few trailing underscores? (leading underscores are banned) if not, could clippy not complain about trailing underscores please?

Tables come up _a lot_ in programming so it would be nice to have good ways to format them.

**Comment by camsteffen** [comment]
I suppose we could have a simple "table heuristic": is this a list of numbers that occupies multiple lines but the first two numbers are on the same line?

**Comment by gilescope** [comment]
Yes it’s a way to make all the numbers the same size in terms of character
length so that cargo fmt aligns them nicely when it wraps them on multiple
lines. It’s the nicest way of doing it with cargo fmt not skipped that I
have found yet...
